src/notation/view/notationpaintview.cpp

```diff
@@ -688,30 +689,19 @@ void NotationPaintView::hoverMoveEvent(QHoverEvent* event)
     }
 }
 
-void NotationPaintView::keyReleaseEvent(QKeyEvent* event)
+void NotationPaintView::shortcutOverride(QKeyEvent* event)
 {
     if (isInited()) {
-        std::string sequence = QKeySequence(event->key()).toString().toStdString();
-        if (event->key() < Qt::Key_Shift && sequence.length() > 0) {
-            if (event->modifiers() & Qt::KeyboardModifier::ShiftModifier) {
-                sequence = "Shift+" + sequence;
-            }
-            if (event->modifiers() & Qt::KeyboardModifier::AltModifier) {
-                sequence = "Alt+" + sequence;
-            }
-            if (event->modifiers() & Qt::KeyboardModifier::ControlModifier) {
-                sequence = "Ctrl+" + sequence;
-            }
-            const auto& shortcuts = shortcutsRegister()->shortcutsForSequence(sequence);
-            const std::shared_ptr<IUiActionsRegister> theRegister = actionsRegister();
-            if (std::find_if(shortcuts.begin(), shortcuts.end(), [theRegister](auto s) {
-                return theRegister->actionState(s.action).enabled;
-            }) != shortcuts.end()) {
-                return;
```

Comment:
there's still an argument for keeping this logic if we decide it makes more sense to have the shortcut-mapped actions to kick in first, but that requires a lot more work to ensure they're only enabled when they make sense (e.g. "next-chord" shouldn't be enabled when a slur is selected etc. etc.)